README.md

```diff
@@ -227,6 +227,7 @@ the following variables:
 - `POSTGRES_PASSWORD`: password used when connecting to the database.
 - `POSTGRES_PORT`: port where the database listens.
 - `POSTGRES_USER`: username used when connecting to the database.
+- `POSTGRES_DATABASE_NAME`: database name (like `homer`).
```

Comment:
Let's preserve the alphabetical order here?

Reply:
Fixed